parcels/field.py

```diff
@@ -168,20 +168,21 @@ def __init__(self, name, data, lon, lat, depth=None, time=None,
         self.time_index_cache = LRUCache(maxsize=2)
 
     @classmethod
-    def from_netcdf(cls, name, dimensions, filenames, **kwargs):
+    def from_netcdf(cls, name, dimensions, filenames, indices, **kwargs):
```

Comment:
I think `indices` needs to have a default value here of an empty dictionary, which should then default to all indices in `read_dimension`, as is the case in `grid`. Some users may load fields after creation of the grid, then add them to the grid themselves.